app/src/components/Navbar.tsx

```diff
@@ -6,14 +6,30 @@ import ThemeSwitch from './ThemeSwitch'
 import clsx from 'clsx'
 import currentModal from '../current-modal'
 import styles from './Navbar.module.css'
+import { useEffect, useState } from 'preact/hooks';
 
 function Navbar() {
   function toggleDownloadModal() {
     currentModal.value = currentModal.value !== 'download' ? 'download' : null
   }
 
+  const [isSticky, setIsSticky] = useState(false);
+
+  useEffect(() => {
+    const handleScroll = () => {
+      const shouldStick = window.scrollY > 0;
+      setIsSticky(shouldStick);
+    };
+
+    window.addEventListener('scroll', handleScroll);
+
+    return () => {
+      window.removeEventListener('scroll', handleScroll);
+    };
+  }, []);
+
```

Comment:
I don't think we need any javascript to make this work.

You can set the header as sticky, but add a `margin-top` on the "editor" enough to push it down.

Reply:
Modified it